Tools-Override/tests.targets

```diff
@@ -246,10 +246,8 @@
   </Target>
 
   <Target Name="CheckTestPlatforms">
-    <GetTargetMachineInfo Condition="'$(TargetOS)' == ''">
-      <Output TaskParameter="TargetOS" PropertyName="TargetOS" />
-    </GetTargetMachineInfo>
     <PropertyGroup>
+      <TargetOS Condition="'$(TargetOS)' == ''">$(DefaultOSGroup)</TargetOS>
```

Comment:
`DefaultOSGroup` is set to `Linux` in cases where `$(OS) == Unix` and the `/Applications/` folder doesn't exist (it should only exist on OSX).